run/logging-manual/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Google LLC. All rights reserved.
+# Use of this source code is governed by the Apache 2.0
+# license that can be found in the LICENSE file.
+
+# Use the official Node.js 10 image.
+# https://hub.docker.com/_/node
+FROM node:10
```

Comment:
@fhinkel is Node 10 fine here, or should we use Node 8?

Reply:
All the Node samples for Cloud Run were built in Node 10. I see it as advantageous to have them be at the forefront of what we're supporting generally. It's also easy to port just one sample to a forward version and see what happens.

Reply:
Node 10 is fine